FreeRTOS/FreeRTOS
# Updating kernel release script to update source files with tags and release in separate scripts

Description
-----------
This PR updates the Kernel release script to separate updating  of source files with version tag and release in two separate scripts. This will enable generation of SBOM after the source files have been updated with version tags.

Test Steps
-----------
Tested on fork

Checklist:
----------



Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
